desktop/extensions-sdk/dev/api/reference/interfaces/Docker.md

```diff
@@ -103,9 +105,9 @@ const images = await ddClient.docker.listImages();
 
 #### Parameters
 
-| Name       | Type  | Description                                                                                                                                                                                                                                                      |
-|:-----------|:------|:-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
-| `options?` | `any` | (Optional). A JSON like `{ "all": true, "filters": JSON.stringify({ dangling: ["true"] }), "digests": true }`  For more information about the different properties see [the Docker API endpoint documentation](../../../../../../engine/api/v1.37.md#tag/Image). |
+| Name | Type | Description |
+| :------ | :------ | :------ |
+| `options?` | `any` | (Optional). A JSON like `{ "all": true, "filters": JSON.stringify({ dangling: ["true"] }), "digests": true * }` * * For more information about the different properties see [the Docker API endpoint documentation](https://docs.docker.com/engine/api/v1.37/#tag/Image). |
```

Comment:
Is this content generated as well? We should probably (I guess we're stripping `docs.docker.com` before we generate the HTML, so probably still okay). If these are not generated, then we should probably make these relative links `/engine/api/<version>/`


I'm wondering why this is referencing API v1.37 - do you know if there's a reason for that? Is the SDK using this version?

v1.37 corresponds with Docker 18.05 (which is long obsolete); https://docs.docker.com/engine/api/#api-version-matrix

Current API version is v1.41 (soon: v1.42).

Reply:
Thx, this is generated from [typedoc comments in the API code](https://github.com/docker/pinata/pull/20250/commits/a68748cceba3996eb7e0cb189e8d5dfc0bd0d92a). I just updated the version number to 1.41. There's no "latest" link I could use to avoid a dependency on a pinned version? (I tried "latest" in the URL instead of the version, not supported :) ). 
The link in typedoc points to docs.docker.com, the generation replaces it with the relative path

Reply:
We do have a `https://docs.docker.com/engine/api/latest/` redirect (which I think is generated when we publish the docs), so that could be an option. Not sure if deep links work with that redirect though 🤔

Reply:
unfortunately no, it does not work for detail pages (Images for example)...

Reply:
Yeah, I doubted that would work. No problem, then we'll just need to keep an eye on updating if we have a new API version (or look if we can improve the redirects to preserve the anchors)